### PR TITLE
Done the changes for 1.2.0 release.

### PR DESCRIPTION
**Pull request by sgantaya**
Done the changes for 1.2.0 release.

### CODE REVIEW

File: NOTICES
Comment (sagdeshp) [inline NOTICES:1]: is this file updated with the latest 1.2.0 version?

**Review by sagdeshp — CHANGES_REQUESTED** [review]

File: build_and_push_custom_image_to_aws_ecr.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+echo "Building and push The TIBCO DeveloperHub custom docker image to AWS ECR"
```
Comment (sagdeshp) [inline build_and_push_custom_image_to_aws_ecr.sh:1]: @sgantaya should we add a README for this file and folder?

File: NOTICES
Comment (sgantaya) [reply NOTICES:1]: yes

File: build_and_push_custom_image_to_aws_ecr.sh (hunk 1 above)
Comment (sgantaya) [reply build_and_push_custom_image_to_aws_ecr.sh:1]: @hpeters83 Can you please answer this, should we add README .

Comment (hpeters83) [reply build_and_push_custom_image_to_aws_ecr.sh:1]: No this is just an example script (like we have example templates)

**Review by sagdeshp — APPROVED** [review]
LGTM

**Pull request merged**